tests/integration/targets/azure_rm_gallery/lookup_plugins/azure_service_principal_attribute.py

```diff
@@ -0,0 +1,92 @@
+# (c) 2018 Yunge Zhu, <yungez@microsoft.com>
```

Comment:
@ephracis you are the author.

Reply:
> @ephracis you are the author.

@xuzhang3 This file was copied from another location and no changes are required to the author. Thank you!

Reply:
Yes, that's true. It would actually be better to put the file somewhere where it could be shared by both integration tests but this was my first attempt at it.

If you have any suggestions on where to put the file and then share it, let me know and I can fix it as part of this PR.

I'm actually wondering why this is not part of the collection itself, and only part of the integration tests. Seems like a good plugin to have. But perhaps there's a reason you don't want it shipped.

Reply:
@ephracis Yes, we are considering it as a plugin that can be applied to the entire repo. Thanks!